plutus-example/plutus-example.cabal

```diff
@@ -0,0 +1,155 @@
+cabal-version: 3.0
+
+name:                   plutus-example
+version:                1.33.0
```

Comment:
What is our versioning policy now that this has moved out of `cardano-node`?  Should it be updated to follow `cardano-node`'s latest version or is this independently versioned now?

Reply:
@asutherlandus